common/src/main/java/com/sun/ts/tests/common/vehicle/jsp/contentRoot/jsp_vehicle.jsp

```diff
@@ -23,7 +23,6 @@
 <%@ page import="java.io.*" %>
 <%@ page import="com.sun.ts.lib.util.*" %>
 <%@ page import="com.sun.ts.lib.harness.*" %>
-<%@ page import="com.sun.ts.lib.porting.*" %>
```

Comment:
Hi @gurunrao 
I had to make this change in this common file which is used across modules. 
Can you please help make sure this will not cause any failures in jdbc, javamail modules.